files/start-strongswan.sh

```diff
@@ -54,6 +54,24 @@ _set_default_variables() {
     export IPSEC_KEYEXCHANGE=${IPSEC_KEYEXCHANGE:-ikev2}
     export IPSEC_ESPCIPHER=${IPSEC_ESPCIPHER:-aes192gcm16-aes128gcm16-ecp256,aes192-sha256-modp3072}
     export IPSEC_IKECIPHER=${IPSEC_IKECIPHER:-aes192gcm16-aes128gcm16-prfsha256-ecp256-ecp521,aes192-sha256-modp3072}
+
+    if [ -z "$IPSEC_REMOTEID" ] && [ "$IPSEC_REMOTEIP" == "%any" ]
```

Comment:
Please avoid bashism `==`. 
`=` or `-eq` is good enough here.

Reply:
Also `-a` inside one `[ ]` (instead of having two `[ ]` linked with `&&`) would save another process.